training/datastore/get_keywords.py

```diff
@@ -29,21 +28,22 @@
 class GetKeywords:
     """This class defines the S3 Connections viz fetching and storing data."""
 
-    def __init__(self, df_=pd.DataFrame(), dict_=dict()):
+    def __init__(self, data_obj, df_=pd.DataFrame(), dict_=dict()):
         """Create an instance for GetKeywords."""
         self.df_ = df_
         self.dict_ = dict_
-        self.get_data = GetData()
+        self.get_data = data_obj
         self.utility = Utility()
 
     def from_existing_df(self, df_, package):
         """Find the keywords from existing dump."""
         if not df_.empty:
             data_lst = df_.loc[df_['name'] == str(package),
-                               ['name', 'description', 'keywords', 'dependencies']].iloc[0]
-            return data_lst
+                               ['name', 'description', 'keywords', 'dependencies']]
+            if not data_lst.empty:
+                return data_lst.iloc[0]
         else:
-            logger.error("Node Package details Dataframe is not existed.")
+            logger.info("Node Package details Dataframe is not existed.")
```

Comment:
`is not existed` -> `does not exist.` This should also be an `error` and not `info` level log since it means the program cannot continue.

Reply:
Actually, if the node details data frame which is the data dump, is not there then It will extract keywords from the NPM registry and Github API.